dissect/target/plugins/os/windows/tasks.py

```diff
@@ -175,6 +182,169 @@
     ],
 )
 
+DailyTriggerRecord = RecordDescriptor(
+    "filesystem/windows/task/trigger/DailyTrigger",
+    [
+        ("uint16", "days_between_triggers"),
+        ("uint16[]", "unused"),
+        ("uint16", "padding"),
+        ("uint16", "reserved2"),
+        ("uint16", "reserved3"),
+    ],
+)
+
+WeeklyTriggerRecord = RecordDescriptor(
+    "filesystem/windows/task/trigger/WeeklyTrigger",
+    [
+        ("uint16", "weeks_between_triggers"),
+        ("string[]", "days_of_week"),
+        ("uint16[]", "unused"),
+        ("uint16", "padding"),
+        ("uint16", "reserved2"),
+        ("uint16", "reserved3"),
+    ],
+)
+
+MonthlyDateTriggerRecord = RecordDescriptor(
+    "filesystem/windows/task/trigger/MonthlyDateTrigger",
+    [
+        ("string", "day_of_month"),
+        ("string[]", "months_of_year"),
+        ("uint16", "padding"),
+        ("uint16", "reserved2"),
+        ("uint16", "reserved3"),
+    ],
+)
+
+MonthlyDowTriggerRecord = RecordDescriptor(
+    "filesystem/windows/task/trigger/MonthlyDowTrigger",
+    [
+        ("string", "which_week"),
+        ("string[]", "day_of_week"),
+        ("string", "months_of_year"),
+        ("uint16", "padding"),
+        ("uint16", "reserved2"),
+        ("uint16", "reserved3"),
+    ],
+)
+
+
+c_atjob = """
```

Comment:
```suggestion
atjob_def = """
```
To be more consistent with other code.